### ISSUE TITLE
PIN for opening new SSH session as a two factor authentication

### ISSUE THREAD

**Issue by alexander-naumov** [alexander-naumov/pam2control#4]
It would be nice to have a possibility to get autogenerated one-time PIN via email. Open new SSH-session for specific user should be possible only after successfull entering this PIN (and after successfull entering SSH password).

**Comment by alexander-naumov** [comment]
Test on FreeBSD 12.1 (x86_64, MTA:sendmail) and openSUSE TW (x86_64, MTA:postfix) is successful.

**Comment by alexander-naumov** [comment]
Test on xUbuntu 20.04 (postfix 3.4.13-0ubuntu1) is OK.

**Comment by alexander-naumov** [comment]
Now p2c is able to generate one-time PIN and send it via email, ask for it (after user provided its SSH password), compare it and create new session or not, depend on user's input.
For generating p2c uses [/dev/urandom(4).](https://man7.org/linux/man-pages/man4/urandom.4.html)

Unfortunately, there is one [PAM_CONV bug ](https://github.com/alexander-naumov/pam2control/issues/17) on xUbuntu system found.
On FreeBSD and openSUSE it works well.
